src/review-form.ts

```diff
@@ -160,6 +210,35 @@ export class ReviewForm extends LitElement {
     `;
   }
 
+  private async setupRecaptcha(): Promise<void> {
+    this.recaptchaWidget = await this.recaptchaManager?.getRecaptchaWidget();
+    this.recaptchaError = false;
+  }
+
+  private async handleSubmit(e: Event): Promise<void> {
+    e.preventDefault();
+    if (!this.reviewForm.reportValidity()) return;
+    if (!this.recaptchaWidget) {
+      this.recaptchaError = true;
+      return;
+    }
+
+    try {
+      const recaptchaToken = await this.recaptchaWidget?.execute();
+      this.dispatchEvent(new Event('recaptchaFinished'));
+
+      if (recaptchaToken) {
```

Comment:
By not using the question mark above, `recaptchaToken` should never be optional so you shouldn't need to check for `recaptchaToken`